packages/xstate-solid/test/useMachine.test.tsx

```diff
@@ -1270,11 +1278,13 @@ describe('useMachine hook', () => {
     });
     const [isAwesome, setIsAwesome] = createSignal(false);
     const App = () => {
-      const [state, send] = useMachine(machine, {
-        guards: {
-          isAwesome: () => isAwesome()
-        }
-      });
+      const [state, send] = useMachine(
```

Comment:
in a separate PR we should remove some of the tests here - we have 2 describe blocks here, one is targeting the strict mode... but Solid doesn't have strict mode. Those tests were just copy-pasted from `@xstate/react` and didn't go through a purge of redundancy :P

Reply:
Sounds good